ui/jest.config.js

```diff
@@ -42,7 +42,6 @@ module.exports = {
       tsConfig: 'tsconfig.test.json'
     }
   },
-  collectCoverage: true,
```

Comment:
turning this off disables a couple of weird messages for `yarn test`, and coverage is collected when running `yarn test:ci`